requirements.txt

```diff
@@ -55,7 +55,7 @@ django-compressor==1.5
 django-statsd-mozilla==0.3.14
 raven==5.2.0
 django-bootstrap-form==3.2
-django-debug-toolbar==1.2.2
+django-debug-toolbar==1.3.2
```

Comment:
django-debug-toolbar was also upgraded here: https://github.com/ccnmtl/ssnm/pull/89 so that PR could be closed or it could be upgraded there instead